x-pack/plugins/observability_solution/synthetics/server/routes/monitor_cruds/get_monitor.ts

```diff
@@ -40,36 +40,36 @@ export const getSyntheticsMonitorRoute: SyntheticsRestApiRouteFactory = () => ({
     response,
     server: { encryptedSavedObjects, coreStart },
     savedObjectsClient,
+    spaceId,
   }): Promise<any> => {
     const { monitorId } = request.params;
     try {
-      const { decrypted } = request.query;
```

Comment:
Was the decrypted param documented for public use or was it just for internal use?

Reply:
it was for internal use.